src/webgpu/api/validation/resource_usages/textureUsageInPassEncoder.spec.ts

```diff
@@ -797,17 +899,40 @@ g.test('bindings_in_bundle')
 g.test('unused_bindings_in_pipeline')
   .params(
     params()
+      // Test compute pass if 'compute' is true. Test render pass otherwise.
+      .combine(pbool('compute'))
       .combine(pbool('useBindGroup0'))
       .combine(pbool('useBindGroup1'))
       .combine(poptions('setBindGroupsOrder', ['common', 'reversed'] as const))
       .combine(poptions('setPipeline', ['before', 'middle', 'after', 'none'] as const))
-      .combine(pbool('callDraw'))
+      .combine(pbool('callDrawOrDispatch'))
   )
   .fn(async t => {
-    const { useBindGroup0, useBindGroup1, setBindGroupsOrder, setPipeline, callDraw } = t.params;
+    const {
+      compute,
+      useBindGroup0,
+      useBindGroup1,
+      setBindGroupsOrder,
+      setPipeline,
+      callDrawOrDispatch,
+    } = t.params;
     const view = t.createTexture({ usage: GPUTextureUsage.STORAGE }).createView();
-    const bindGroup0 = t.createBindGroup(0, view, 'readonly-storage-texture', '2d', 'rgba8unorm');
-    const bindGroup1 = t.createBindGroup(0, view, 'writeonly-storage-texture', '2d', 'rgba8unorm');
+    const bindGroup0 = t.createBindGroup(
+      0,
+      view,
+      'readonly-storage-texture',
+      '2d',
+      'rgba8unorm',
+      false
```

Comment:
Needs to be `compute`?

Reply:
Oops. My bad. I must be sleepy at that time... Done.